yard-api/src/main/java/org/kubesmarts/yard/api/model/DecisionLogic.java

```diff
@@ -0,0 +1,23 @@
+package org.kubesmarts.yard.api.model;
+
+import jakarta.json.bind.annotation.JsonbSubtype;
+import jakarta.json.bind.annotation.JsonbTypeInfo;
```

Comment:
I was expecting `git mv` to actively track as a move but instead we got a deletion and an add. Have you used `git mv` or did it throughout Intelij Idea?

Reply:
I used folder rename in Idea. I can check if move does it in a nicer way.